<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryWindow</class>
 <widget class="QMainWindow" name="MemoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="StatusCircle" name="wRamCircle" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>RAM total</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbRamTotal">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>RAM free</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbRamAvail">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Swap total</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbSwapTotal">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Swap free</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbSwapFree">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>1</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>1</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LayoutDirection::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="StatusCircle" name="wMemCircle" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Root path</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbMemRootPath">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Device</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbMemDevice">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Filesystem</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbMemFileSysType">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Total Kbytes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbMemTotal">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>18</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>Free Kbytes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbMemAvail">
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 170, 0);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>---------</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_27">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>KB</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnMemBack">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="button.qrc">
           <normaloff>:/backbutton/backbutton.png</normaloff>:/backbutton/backbutton.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusCircle</class>
   <extends>QWidget</extends>
   <header location="global">StatusCircle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="button.qrc"/>
 </resources>
 <connections/>
</ui>
